cnxdb/archive-sql/get-books-containing-page.sql

```diff
@@ -0,0 +1,25 @@
+-- ###
+-- Copyright (c) 2013, Rice University
+-- This software is subject to the provisions of the GNU Affero General
+-- Public License version 3 (AGPLv3).
+-- See LICENCE.txt for details.
+-- ###
+
+-- arguments: page_uuid:string; page_version:string
```

Comment:
Thank you for including this. We aren't currently using it, but the hope is that it will be useful for generating functions associated with these sql files.